bin/regrade.py

```diff
@@ -88,6 +88,14 @@ def main():
                 my_semester=my_dirs[len(data_dirs)]
                 my_course=my_dirs[len(data_dirs)+1]
                 my_gradeable=my_dirs[len(data_dirs)+3]
+                gradeable_config = os.path.join(data_dir,my_semester,my_course,"config/build/"+"build_"+my_gradeable+".json")
+                with open(gradeable_config, 'r') as build_configuration:
+                    datastore = json.load(build_configuration)
+                    required_capabilities = datastore.get('required_capabilities', 'default')
+                    max_grading_time = datastore.get('max_possible_grading_time', -1)
+
+                #get the current time
+                queue_time = dateutils.write_submitty_date() + " " + str(dateutils.get_timezone())
```

Comment:
Leave off the get_timezone thing.  All that's needed is in the write_submitty_date.